libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -0,0 +1,237 @@
+                />
+              </Pivot>
+            </Stack.Item>
+            <Stack.Item>
+              <CommandBarButton
+                className={classNames.filterButton}
+                iconProps={{ iconName: "Settings" }}
+                text={localization.InterpretVision.Dashboard.settings}
+              />
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        <Stack.Item>
+          <div className={classNames.line} />
+        </Stack.Item>
+        <Stack.Item>
+          <Stack>
+            <Stack.Item className={classNames.cohortPickerLabelWrapper}>
+              <Text
+                variant="mediumPlus"
+                className={classNames.cohortPickerLabel}
+              >
+                {localization.Interpret.ModelPerformance.cohortPickerLabel}
+              </Text>
+            </Stack.Item>
+            <Stack.Item>
+              <Stack
+                horizontal
+                className={classNames.toolBarContainer}
+                tokens={{ childrenGap: "l1" }}
+                verticalAlign="center"
+              >
+                <Stack.Item>
+                  <Dropdown
+                    className={classNames.cohortDropdown}
+                    id="dataExplorerCohortDropdown"
+                    options={DropdownOptions}
+                    selectedKey={0}
+                  />
+                </Stack.Item>
+                <Stack.Item>
+                  <SearchBox
+                    className={classNames.searchBox}
+                    placeholder="Search"
+                  />
+                </Stack.Item>
+                <Stack.Item>
+                  <CommandBarButton
+                    className={classNames.filterButton}
+                    iconProps={{ iconName: "Filter" }}
+                    text={localization.InterpretVision.Dashboard.filter}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        <Stack.Item>
+          <Stack
+            horizontal
+            horizontalAlign="space-between"
+            verticalAlign="start"
+          >
+            <Stack.Item>
+              <Slider
+                max={80}
+                min={20}
+                className={classNames.slider}
+                label={localization.InterpretVision.Dashboard.thumbnailSize}
+                defaultValue={40}
+                showValue={false}
+                onChange={this.onSliderChange}
+              />
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        {this.state.selectedKey === VisionDatasetExplorerTabOptions.First && (
+          <Stack horizontal={false} grow tokens={{ childrenGap: "l1" }}>
+            <Stack
+              horizontal
+              tokens={{ childrenGap: "l1" }}
+              style={{
+                height:
+                  this.state.imageDim * 3 + Math.floor(this.state.imageDim / 2)
```

Comment:
can you add comment on how this  calculation is made and how the images are rendered?